system-addon/content-src/styles/_variables.scss

```diff
@@ -70,10 +70,12 @@ $wrapper-default-width: $grid-unit * 2 + $base-gutter * 1; // 2 top sites
 $wrapper-max-width-small: $grid-unit * 3 + $base-gutter * 2; // 3 top sites
 $wrapper-max-width-medium: $grid-unit * 4 + $base-gutter * 3; // 4 top sites
 $wrapper-max-width-large: $grid-unit * 6 + $base-gutter * 5; // 6 top sites
-$break-point-small: $wrapper-max-width-small + $base-gutter * 2;
-$break-point-medium: $wrapper-max-width-medium + $base-gutter * 2;
-$break-point-large: $wrapper-max-width-large + $base-gutter * 2;
-$wrapper-card-outer: 768px;
+$wrapper-max-width-widest: $grid-unit * 8 + $base-gutter * 7; // 8 top sites
+$scrollbar-width: 16px;
```

Comment:
A comment about why we need this and that we're just using a single value for all os